build.gradle

```diff
@@ -241,7 +241,7 @@ subprojects {
     }
   }
 
-  def shouldUseJUnit5 = ["tools", "raft", "log4j-appender"].contains(it.project.name)
+  def shouldUseJUnit5 = ["tools", "raft", "log4j-appender", "examples"].contains(it.project.name)
```

Comment:
Is "examples" the right name? Given that you converted `streams-examples`?

Reply:
> Is "examples" the right name? Given that you converted streams-examples?

gradle uses the folder name as project name. It can be override if we want to make more readable naming. All project names are shown below.

```
clients
connect
core
examples
generator
jmh-benchmarks
log4j-appender
raft
streams
tools
api
basic-auth-extension
file
json
mirror
mirror-client
runtime
transforms
examples
streams-scala
test-utils
upgrade-system-tests-0100
upgrade-system-tests-0101
upgrade-system-tests-0102
upgrade-system-tests-0110
upgrade-system-tests-10
upgrade-system-tests-11
upgrade-system-tests-20
upgrade-system-tests-21
upgrade-system-tests-22
upgrade-system-tests-23
upgrade-system-tests-24
upgrade-system-tests-25
upgrade-system-tests-26
upgrade-system-tests-27
```